### ISSUE TITLE
AMF faulty

### ISSUE THREAD

**Issue by gzcwnk** [NillerMedDild/Enigmatica2Expert#1181]
**Modpack version**
1.60

**Can the issue be reproduced?**
unknown

**Logs**
Input pastebin link to crash-report or latest log, if applicable

**Issue**
Advanced metallurgical fabricator does not function even in creative mode.  Keeps saying Status: item input missing.  

If this is a feature request, feel free to delete the template.

**Comment by NillerMedDild** [comment]
It works just fine, try crafting something. Missing input item usually just means you're not trying to craft anything.

**Comment by gzcwnk** [comment]
uh no its not crafting at all..

**Comment by NillerMedDild** [comment]
Still, it works just fine. What are you trying to make, and which ingredients are you giving it? Keep in mind most recipes require you to input 100.000 RF/t.

**Comment by gzcwnk** [comment]
I tried to make machine frames, and other things.   I am feeding it with an advanced power cell via an elite universal cable

**Comment by gzcwnk** [comment]
from online videos the energy supply looks big enough, ie ive matched it.  Does not work in actual game nor creative mode

**Comment by gzcwnk** [comment]
fed with creative powercell upstream

**Comment by NillerMedDild** [comment]
Screenshots of the machine's inputs would be helpful, but I recommend you take this issue to the Discord, as it's not a bug.

**Comment by gzcwnk** [comment]
tried discord they have no idea

**Comment by gzcwnk** [comment]
https://imgur.com/a/Xsd79C5

**Comment by NillerMedDild** [comment]
I mean the machine's input ports. I need to see the ingredients you're giving it, the tier of inputs, and the expected output.

**Comment by NillerMedDild** [comment]
and the GUI of the controller, please.

**Comment by gzcwnk** [comment]
https://imgur.com/a/Xsd79C5

**Comment by NillerMedDild** [comment]
and the rest?

**Comment by gzcwnk** [comment]
https://imgur.com/a/sTJT0CF

**Comment by gzcwnk** [comment]
https://imgur.com/a/QS2vs5R

**Comment by gzcwnk** [comment]
https://imgur.com/a/xEFMJ6L

**Comment by NillerMedDild** [comment]
And the Fluid Input Port? Can it hold the amount of fluid the recipe needs?

**Comment by gzcwnk** [comment]
its a huge port holding 20000mb of liquid glass

**Comment by gzcwnk** [comment]
or big  = 20000 i tried huge = 40000 in creative

**Comment by gzcwnk** [comment]
https://imgur.com/a/wzF8u2i

**Comment by NillerMedDild** [comment]
Cool. And the energy input is receiving 50.000RF/t or more?

**Comment by NillerMedDild** [comment]
If that still doesn't work, I suggest you update to 1.62 and try again.

**Comment by gzcwnk** [comment]
I have a blue advanced energy cell, youtube videos were using this, but I even put a creative energy cell in.  I will try an upgrade.

**Comment by NillerMedDild** [comment]
But does the Energy Cell have sufficient power?

**Comment by gzcwnk** [comment]
I dont understand, the advanced energy cell is full and the creative energy cell is always full?   Inside the machine, the energy bar is 100%, if it was empty then yes that would be a clue. I mean maybe I should raise a request for better docs for the mod? its giving me no idea on what's wrong.

**Comment by gzcwnk** [comment]
Also can we have a README in the server zip saying what to delete before an upgrade? or a URL?  I am told something exists but I cant find it.

**Comment by gzcwnk** [comment]
OK, on 1.62 no difference still does not work.

**Comment by NillerMedDild** [comment]
Yes, I'm adding a readme to the server files. 

Please try breaking and replacing the structure.

**Comment by gzcwnk** [comment]
Cool thanks.

Upgraded to 1.62 OK, thanks runs great. I have tried breaking it to nothing already no change, plus changing out the inputs for bigger and smaller ones.  I strongly suspect I am making an error due to the existing info / tutorials I found is either obsolete or simply wrong for the power input.   I will have to dig further tonight but I have bypassed it with an excavator and bauxite to get titanium anyway.
